server/plugin/api.go

```diff
@@ -287,7 +287,7 @@ func (p *Plugin) connectUserToGitHub(c *Context, w http.ResponseWriter, r *http.
 		return
 	}
 
-	appErr := p.API.KVSetWithExpiry(state.Token, stateBytes, TokenTTL)
+	appErr := p.API.KVSetWithExpiry(state.Token+githubOauthKey, stateBytes, TokenTTL)
```

Comment:
Should be `githubOauthKey` a prefix, as the other keys in the store are?

Reply:
I agree with ☝️